webbpsf/webbpsf_core.py

```diff
@@ -1822,8 +1823,10 @@ def __init__(self):
         self._rotation = 4.83544897  # V3IdlYAngle, Source: SIAF PRDOPSSOC-059
                                 # This is rotation counterclockwise; when summed with V3PA it will yield the Y axis PA on sky
 
-        self.options['pupil_shift_x'] = -0.0069 # CV3 on-orbit estimate (RPT028027) + OTIS delta from predicted (037134)
-        self.options['pupil_shift_y'] = -0.0027
+        # Coordinate system note: The pupil shifts get applied at the instrument pupil, which is an image of the OTE exit pupil
+        # and is thus flipped in Y relative to the V frame entrance pupil. Therefore flip sign of pupil_shift_y
+        self.options['pupil_shift_x'] = -0.0068  # In flight measurement. See Wright, Sabatke, Telfer 2022, Proc SPIE 
+        self.options['pupil_shift_y'] = -0.0110  # Sign intentionally flipped relative to that paper!! See note above.
```

Comment:
For my own clarification. In the paper the values are V2 = +0.0068 and V3 = -0.0110, this is the measured shear projected to the OTE Exit pupil. However, this is the relative shear from the primary mirror to the pupil and webbpsf needs the pupil shear. Therefore, we change the sign of both values, -0.0068 and 0.0110 and then change the y shift sign because it is applied at the instrument pupil. Thus, the final numbers, -0.0068 and -0.0110.

Reply:
👍 . And you saw the emails with Randal asking him to help confirm I had the signs right here.

Reply:
oh you are right, I forgot about the email. Yes, you explained everything there.